implementations/elixir/ockam/ockam/mix.exs

```diff
@@ -72,6 +74,7 @@ defmodule Ockam.MixProject do
 
   defp aliases do
     [
+      credo: "credo --strict",
```

Comment:
there's another alias for this below `lint.credo`

Reply:
Sure, I noticed that. However, I was running `mix credo` automatically in VSCode and did not look at the alias list.  It seems like switching the default to what we expect would keep others from falling into the same trap.  I'd argue that we should do that on all of our elixir packages, too - but didn't think about it til just now.

Reply:
cool, makes sense 👍